src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/memory.cpp

```diff
@@ -6,10 +6,64 @@
 
 #include <vector>
 
+#include "ngraph/opsets/opset11.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "openvino/op/util/variable.hpp"
+
 using namespace LayerTestsDefinitions;
+using namespace ngraph;
+using namespace opset11;
 
 namespace {
 
+class MemoryTestGna : public MemoryTest {
+    using Super = MemoryTest;
+
+public:
+    InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo& info) const override {
+        // data comparison against relative threshold works correct when input data is shifted away from 0.
+        // This happens also when POT generated FQ layers are added (see below) to original model.
+        return FuncTestUtils::createAndFillBlob(info.getTensorDesc(), 5, 5);
```

Comment:
does it  mean that input values are from range 5 to 10?

Reply:
yes